package.json

```diff
@@ -100,7 +101,8 @@
     "jest": "27.2.1",
     "nock": "13.0.11",
     "prettier": "npm:wp-prettier@2.0.5",
-    "publish-please": "5.5.2"
+    "publish-please": "5.5.2",
+    "rimraf": "^3.0.2"
```

Comment:
```suggestion
    "rimraf": "3.0.2"
```

Reply:
Good point, just pushed up a fix to pin that version